app/models/edition.rb

```diff
@@ -92,8 +92,8 @@ class Edition < ApplicationRecord
       .find_by!(find_by)
   end
 
-  def self.create_initial(document, user = nil, tags = {})
-    revision = Revision.create_initial(document, user, tags)
+  def self.create_initial(document, user = nil, tags = {}, document_type_id = "")
```

Comment:
I don't think we'd want this to default to an empty string. I think we expect this to actually have a value so may want to make it a required parameter.

As we've now got 4 params we may well want to make this keyword arguments.

Reply:
done!